### ISSUE TITLE
Single/Multi select is not supported in editors

### ISSUE THREAD

**Issue by abhishekp21** [nocode-js/sequential-workflow-designer#41]
Hi, I wanted to express my appreciation for this excellent library you've created!

However, I have encountered a situation where I must include a single/multi-select dropdown in the step editor. I attempted to use mat-select and mat-option to display the dropdown in the user interface. While the dropdown is visible, the data is not binding properly, and the dropdown is not populating with any values.

![image](https://user-images.githubusercontent.com/18423249/236675252-7f306c0f-8f5e-446e-bb00-f4f9b814b3eb.png)

![image](https://user-images.githubusercontent.com/18423249/236675144-680ac122-ac6e-44be-bad3-c1a024fdf8b5.png)

![image](https://user-images.githubusercontent.com/18423249/236675220-60e89ab6-0b33-4c28-9766-c2064213e80d.png)

Is there something I'm missing ?

**Comment by b4rtaz** [comment]
Do you have the same problem if you use a [standard select](https://stackoverflow.com/a/35945293/10828660)?

**Comment by abhishekp21** [comment]
@b4rtaz It works fine if we use standard select but problem exist if we use mat-select & mat-options.

**Comment by abhishekp21** [comment]
The single select dropdown works well, but for multi-select functionality, using the standard select with multiple attribute is not convenient because it's difficult to track which options are selected. Thus, I prefer to use a dropdown with checkboxes. To implement this, I created a separate component in Angular and tried to use it in my project. However, when I tried to use it in the editor, the dropdown did not display properly. You can see an example of component build in the following link: https://stackblitz.com/edit/multiselect-checkbox-dropdown-kftztx?file=src%2Fapp%2Fapp.component.ts.

![image](https://user-images.githubusercontent.com/18423249/236696519-01675f77-7871-4412-a458-2617413c3136.png)

![image](https://user-images.githubusercontent.com/18423249/236696479-c0666aa1-5a09-4b0b-a58e-bf466b926105.png)

**Comment by b4rtaz** [comment]
I've tried to reproduce the problem and I noticed that some features of the Material package work fine inside the designer (like buttons), some don't. It looks like the Material package doesn't work properly if we render DOM by the `createEmbeddedView` method. So this is more the bug of the Material package than the SWD.

![angular-material-err](https://github.com/nocode-js/sequential-workflow-designer/assets/12797776/d8416c5d-9b0f-45ac-b494-90c375c314fd)
